calicoctl/calicoctl.go

```diff
@@ -48,7 +49,7 @@ Options:
 	if logLevel := arguments["--log-level"]; logLevel != nil {
 		parsedLogLevel, err := log.ParseLevel(logLevel.(string))
 		if err != nil {
-			fmt.Printf("Unknown log level: %s, expected one of: \n" +
+			fmt.Printf("Unknown log level: %s, expected one of: \n"+
```

Comment:
not sure I understand what you mean. The spacing was adjusted by `gofmt` (which should have been to begin with)